util/feeding.py

```diff
@@ -29,6 +33,14 @@ def read_csvs(csv_files):
             source_data = source_data.append(file)
     return source_data
 
+def augment(sample, sr):
+    sample = librosa.core.to_mono(np.transpose(sample, (1, 0)))
+    if random.random() < 0.5:
```

Comment:
Why this magic number 0.5?

I'd guess it'd be best if this were the default value of a flag,
as we don't know if this is optimal. In addition it's very likely
this will have to change from data set to data set.

Reply:
Will add the flag. Lets assume i add the flag X for X% pitch modulation, then it would be (1-X)% for time stretch in this implentation. Is that ok?